Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp

```diff
@@ -228,6 +228,10 @@ namespace AZ
             //Load device features now that we have loaded all extension info
             physicalDevice.LoadSupportedFeatures();
 
+            DEVMODE DisplayConfig;
```

Comment:
Looks like this part needs to be PAL-ified for Linux support (I added a very thin PAL layer for RHI recently you should be able to use)

Reply:
I PAL-ified within the Vulkan backend itself.